pkg/eks/client.go

```diff
@@ -61,7 +61,10 @@ func (c *Client) new(spec *api.ClusterConfig, stsClient stsiface.STSAPI) (*Clien
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to create API client configuration from client config")
 	}
+
 	c.rawConfig = rawConfig
+	c.rawConfig.QPS = float32(25)
+	c.rawConfig.Burst = int(c.rawConfig.QPS) * 2
```

Comment:
`QPS`=`queries per second`. Setting it to max 25, it normally defaults to 5. The library defaults `Burst` to 10, so I've followed the pattern of setting `Burst` to 2x the `QPS`.

LMK if this doesn't make sense!

Reply:
Am I dumb, how does this relate to parallel node draining?

Reply:
prior to this you could only make 5 k8s queries per second, so I was getting massively throttled when running more than 5 drains in parallel.

Reply:
Ah gotcha